scripts/mobile-snapshot-diff.py

```
@@ -0,0 +1,193 @@
+    return json.loads(raw).get("workspaces", [])
+
+
+def selected_workspace_and_terminal():
+    for ws in list_workspaces():
+        if ws.get("is_selected"):
+            term = (ws.get("terminals") or [None])[0]
+            return ws["id"], (term or {}).get("id")
+    raise RuntimeError("no selected workspace")
+
+
+def mac_visible_lines(workspace_id: str, terminal_id: str, lines: int = 45):
```

Ruff (0.15.14)

[warning] Unused function argument: `lines`

(ARG001)